packages/website/docs/MultiSelect.md

```diff
@@ -1,23 +1,50 @@
-## Guidelines
+## Best Practices
 
-### General
+Use multi-select to allow user to select options from a **long list that can be filtered**. Multi-selects are more appropriate when the available list of options is very long or when the space is limited.
 
--   If space allows, and there are no more than 7 options in total, consider using [checkboxes](#/components/Checkbox) instead.
--   Keep option labels as short as possible in order to limit width to a minimum.
--   When an option is selected, it should be removed from the options list.
+Use alphanumerical order unless a more suited ordering rational applies (such as sizes or security levels).
 
-### Options Sorting
+Always include the ability to filter the list when there are over 20 options.
 
--   Always put the most common items at the top of the list. If no option is more common than the others, use alphanumeric or chronological order.
+## Labeling
 
-### Label
+Keep titles and labels short, **under one line**.
 
--   In most cases, you should include a label to identify the component.
+### Title
+
+The title is used to indicate the type of information to be provided.
+
+Use descriptive labels without action verbs.
+
+-   E.g. Use "Grocery items" instead of "Select all items for your grocery".
+
+### Labels
+
+Labels are used to name each option. The width of the input should allow to fully display the name of the chosen option.
+
+Use a consistent writing style for all options in the list.
+
+## Help text and instructions
+
+The placeholder text should indicate the type of information that must be selected with an action verb.
+
+-   E.g. Use "Select a condiment" instead of "Select an option".
```

Comment:
```suggestion
The placeholder text should indicate the type of information to select. Use an action verb. For example, write "Select a condiment" rather than "Select an option".
```